tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -6656,48 +6700,56 @@ <h2>Show similarity to selected datapoint</h2>
           for (let i = 0; i < attributions.indices.length; i++) {
             const idx = attributions.indices[i];
             const datapoint = Object.assign({}, this.visdata[idx]);
-            // TODO(jwexler): Support attributions from multiple models.
-            // For now, we only display attribution from the first model, if WIT
-            // is loaded with two models.
-            const attribs =
-              attributions.attributions[0] == null
-                ? {}
-                : attributions.attributions[0][i];
-            const keys = Object.keys(attribs);
-            for (let j = 0; j < keys.length; j++) {
-              // If the attributions for a key is a 2D array then treat the first
-              // entry as sparse keys for attributions and the second entry as
-              // the attributions for the sparse keys in the first array.
-              // Otherwise the attribution is a single value or list and can be
-              // used as such.
-              let attribKey = attributionPrefix + keys[j];
-              if (
-                Array.isArray(attribs[keys[j]]) &&
-                attribs[keys[j]].length == 2 &&
-                Array.isArray(attribs[keys[j]][0])
-              ) {
-                const attribSparseKeyKey =
-                  attributionPrefix + keys[j] + attributionSparseKeySuffix;
-                datapoint[attribSparseKeyKey] = attribs[keys[j]][0];
-                attribKey =
-                  attributionPrefix + keys[j] + attributionSparseValueSuffix;
-                datapoint[attribKey] = attribs[keys[j]][1];
-              } else {
-                datapoint[attribKey] = attribs[keys[j]];
-              }
-              let newAttributions = Array.isArray(datapoint[attribKey])
-                ? datapoint[attribKey]
-                : [datapoint[attribKey]];
-              for (
-                let attribIdx = 0;
-                attribIdx < newAttributions.length;
-                attribIdx++
-              ) {
-                if (newAttributions[attribIdx] > this.maxAttribution) {
-                  this.maxAttribution = newAttributions[attribIdx];
+            for (
+              let modelNum = 0;
```

Comment:
In the case model 2 has attributions and model 1 does not, WIT may falsely show model 2's attributions as model 1's. We can hide the model indicator whenever the attribution length is 1 for now and add a TODO to handle this case correctly with the indicator in future.

Reply:
implemented this by only appending model index to these new datapoint values when both models have attributions, through new method attributionStrWithModelName_ and correct hiding of the attribution model selector. added TODO for further work.